src/d3a/models/strategy/predefined_load.py

```diff
@@ -125,6 +125,11 @@ def _allowed_operating_hours(self, time):
         return True
 
     def area_reconfigure_event(self, **kwargs):
+        """Reconfigure the device properties at runtime using the provided arguments.
+
+        This method is triggered when the device strategy is updated while the simulation is
+        running. The update can happen via live events (triggered by the user) or scheduled events.
+        """
```

Comment:
Every strategy kind as well as the area has this function. I am not sure how to handle it. Either you add this docstring to all of them or just leave it. What do you think?

Reply:
I found something that can help us here in [PEP 257](https://www.python.org/dev/peps/pep-0257/):
>The docstring for a class should summarize its behavior and list the public methods and instance variables. If the class is intended to be subclassed, and has an additional interface for subclasses, this interface should be listed separately (in the docstring). The class constructor should be documented in the docstring for its __init__ method. Individual methods should be documented by their own docstring.
>
> If a class subclasses another class and its behavior is mostly inherited from that class, its docstring should mention this and summarize the differences. Use the verb "override" to indicate that a subclass method replaces a superclass method and does not call the superclass method; use the verb "extend" to indicate that a subclass method calls the superclass method (in addition to its own behavior).

So I can move the full docstring into the `BaseStrategy` definition of the method, and just keep the first line in the child. What do you think? Leaving it without docstring would raise a pylint warning.

Reply:
OK! Agreed.